gunicorn/config.py

```diff
@@ -538,9 +538,9 @@ class Bind(Setting):
     validator = validate_list_string
 
     if 'PORT' in os.environ:
-        default = ['0.0.0.0:{0}'.format(os.environ.get('PORT'))]
+        default = ['[::]:{0}'.format(os.environ.get('PORT'))]
```

Comment:
At least on my machine, this returns only IPv6 addresses from `socket.getaddrinfo()`. Since this is a list, maybe we want to give it two entries so Gunicorn binds to IPv4 and IPv6 by default?

Reply:
If you bind to :: you will bind to any address  (IPv4 and IPv6) as long the IPV6_V6ONLY flag is not set.